ui/components/multichain/app-header/meta-fox-logo.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
+import { useSelector } from 'react-redux';
+///: END:ONLY_INCLUDE_IF
+import { useHistory } from 'react-router-dom';
+import { DEFAULT_ROUTE } from '../../../helpers/constants/routes';
+
+import {
+  AlignItems,
+  Display,
+  JustifyContent,
+} from '../../../helpers/constants/design-system';
+import { Box } from '../../component-library';
+import {
+  ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
+  getTheme,
+  ///: END:ONLY_INCLUDE_IF
+} from '../../../selectors';
```

Comment:
```suggestion
///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
import { getTheme } from '../../../selectors';
///: END:ONLY_INCLUDE_IF
```

suggestion avoids blank import